stdlib/public/core/Character.swift

```diff
@@ -120,7 +120,7 @@ public struct Character :
   @warn_unused_result
   static func _smallSize(value: UInt64) -> Int {
     var mask: UInt64 = 0xFF
-    for var i = 0; i < 8; i += 1 {
+    for i in 0..<8 {
```

Comment:
Is there an official style guide regarding these loops?
I find it easier to read when it's in the style of `for i in 0 ..< 8` (with that spacing)